prow/cmd/status-reconciler/main.go

```diff
@@ -53,6 +54,15 @@ type options struct {
 
 	tokenBurst    int
 	tokensPerHour int
+
+	// The following are used for reading/writing to GCS.
+	gcsCredentialsFile string
```

Comment:
Are these two and their associated flags something that could be DRYed out into a struct and embedded in all the controllers that do something similar? (tide, deck, etc)

Reply:
@adshmh I think we forgot to expose this as a flag in the first place :man_facepalming: 

Perhaps we can fix that with the patch to share this code between components that mean to write to GCS?

Reply:
Thanks for catching that. Yes, I will include that in the patch.